src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,21 @@
+const jwtService = require('../services/jwt.service');
+const { ApiError } = require('../utils/api.error');
+
+function authMiddleware(req, res, next) {
+  const authHeader = req.headers['authorization'] || '';
+  const [, accessToken] = authHeader.split(' ');
+
+  if (!authHeader || !accessToken) {
+    throw ApiError.unauthorized();
+  }
+
+  const userData = jwtService.validateAccessToken(accessToken);
```

Comment:
Consider handling potential errors from `jwtService.validateAccessToken` to prevent unhandled promise rejections.